amd/src/options.js

```diff
@@ -60,34 +73,45 @@ define(['jquery', 'core/modal_factory', 'core/url'], function ($, modalfactory,
                 } else {
                     unix = -1;
                 }
-                s = url.relativeUrl("/admin/tool/messenger/predict_affected_users_ajax.php",
-                    {'roles': roles.join(","), 'knockoutdate': unix}, true);
+                // Build url.
+                params = {'roles': roles.join(","), 'knockoutdate': unix};
             }
-            var xhttp = new XMLHttpRequest();
-            xhttp.onreadystatechange = function () {
-                if (this.readyState === 4 && this.status === 200) {
-                    $('#predictionbox').text(this.responseText);
-                }
-            };
-            xhttp.open("GET", s);
-            xhttp.send();
+            ajax.call([
+                {methodname: 'tool_messenger_predict_users', args: params}
```

Comment:
The prediction call is returning an error. The knockoutdate seems to be empty.

Reply:
This works for me can you tell me what parameters you are using?